src/components/WalletTabs.vue

```diff
@@ -6,15 +6,12 @@ import { useI18n } from "vue-i18n";
 import SolanaLogoURL from "@/assets/solana-dark.svg";
 import SolanaLightLogoURL from "@/assets/solana-light.svg";
 import { AccountMenu, AccountMenuList, AccountMenuMobile } from "@/components/nav";
-import { requireLoggedIn } from "@/modules/auth";
 import ControllerModule from "@/modules/controllers";
 import { i18n, setLocale } from "@/plugins/i18nPlugin";
 import { NAVIGATION_LIST } from "@/utils/enums";
 
 import LanguageSelector from "./nav/LanguageSelector.vue";
 
-requireLoggedIn();
```

Comment:
A user can access any of `/wallet` routes with redirect flow params without login. (As login checks are skipped in router for redirect flow). Not necessarily a security issue but lets keep the check here?

Reply:
shouldn't the checks be in route(or auth service), and not in the components?
Let's get on a call for this whenever you find time.

Reply:
my issue is that 
1. I do not want to keep same code at 2 places [nft.vue and base.vue]
2. authentication based routing should be only done at the router.
I'll reformat this in a parallel PR.